src/Avalonia.Themes.Default/Controls/SplitButton.xaml

```diff
@@ -0,0 +1,430 @@
+﻿<Styles xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        x:CompileBindings="True">
+
+  <!-- This style is heavily based on the Fluent v1 style.
+       The base resources are switched out along with a few minor changes such as Padding, CornerRadius. -->
+
+  <Design.PreviewWith>
+    <Border Padding="20">
+      <SplitButton />
+    </Border>
+  </Design.PreviewWith>
+
+  <Styles.Resources>
+    <x:Double x:Key="SplitButtonPrimaryButtonSize">32</x:Double>
+    <GridLength x:Key="SplitButtonSecondaryButtonSize">32</GridLength>
+    <Thickness x:Key="SplitButtonBorderThemeThickness">1</Thickness>
+    
+    <StaticResource x:Key="SplitButtonBackground" ResourceKey="ThemeControlMidBrush" />
```

Comment:
I think you can remove this resources in between and use "theme*" resources directly.

Reply:
That requires changes many places in the styles. The reason it was not done this way is simplicity and time. Changes can easily be copy pasted in the future using the resource-level abstraction. 

I could easily make the same comment about Fluent styles themselves. There the extra resource abstraction also allows lightweight styling changes.

I think you are creating more work for little performance benefit here.